java/integTest/src/test/java/glide/cluster/ClientTests.java

```diff
@@ -0,0 +1,272 @@
+import glide.api.models.ClusterValue;
+import glide.api.models.commands.InfoOptions;
+import glide.api.models.configuration.NodeAddress;
+import glide.api.models.configuration.RedisClusterClientConfiguration;
+import glide.api.models.configuration.RedisCredentials;
+import glide.api.models.exceptions.ClosingException;
+import glide.api.models.exceptions.RequestException;
+import java.lang.module.ModuleDescriptor;
+import java.security.SecureRandom;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+import java.util.stream.Collectors;
+import lombok.SneakyThrows;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
+
+public class ClientTests {
+    private RedisClusterClientConfiguration.RedisClusterClientConfigurationBuilder<?, ?>
+            commonClusterClientConfig() {
+        return RedisClusterClientConfiguration.builder()
+                .address(NodeAddress.builder().port(CLUSTER_PORTS[0]).build());
+    }
+
+    private static Map<String, String> parseInfoResponse(ClusterValue<String> infoResponse) {
+        String value;
+        if (infoResponse.hasSingleData()) {
+            value = infoResponse.getSingleValue();
+        } else {
+            value = infoResponse.getMultiValue().entrySet().iterator().next().getValue();
+        }
+
+        return value
+                .lines()
+                .filter(line -> line.contains(":"))
+                .map(line -> line.split(":", 2))
+                .collect(
+                        Collectors.toMap(
+                                parts -> parts[0],
+                                parts -> parts[1],
+                                (existingValue, newValue) -> newValue,
+                                HashMap::new));
+    }
+
+    private static String getRandomString(int length) {
+        String asciiLetters = "ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz";
+        SecureRandom random = new SecureRandom();
+        StringBuilder sb = new StringBuilder(length);
+
+        for (int i = 0; i < length; i++) {
+            int index = random.nextInt(asciiLetters.length());
+            char randomChar = asciiLetters.charAt(index);
+            sb.append(randomChar);
+        }
+
+        return sb.toString();
+    }
+
+    @SneakyThrows
+    private Boolean check_if_server_version_gte(RedisClusterClient client, String minVersion) {
+        ClusterValue<String> infoStr = client.info(InfoOptions.builder().section(SERVER).build()).get();
+        String redisVersion = parseInfoResponse(infoStr).get("redis_version");
+        assertNotNull(redisVersion);
+        return ModuleDescriptor.Version.parse(redisVersion)
+                        .compareTo(ModuleDescriptor.Version.parse(minVersion))
```

Comment:
You can simplify if you fix the import
```suggestion
        return Version.parse(redisVersion).compareTo(Version.parse(minVersion))
```